DependencyInjection/SwiftmailerExtension.php

```diff
@@ -157,6 +160,15 @@ protected function configureMailerTransport($name, array $mailer, ContainerBuild
             $container
                 ->setDefinition(sprintf('swiftmailer.mailer.%s.transport.buffer', $name), $bufferDecorator);
 
+            $configuratorDecorator = new DefinitionDecorator('swiftmailer.configurator.abstract');
+            $container
+                ->setDefinition(sprintf('swiftmailer.configurator.transport.%s', $name), $configuratorDecorator)
+                ->setArguments(array(
+                    '%swiftmailer.mailer.' . $name . '.transport.smtp.local_domain%',
```

Comment:
Symfony CS does not use spaces around dots, and in such situations, we prefer to use `sprintf` (the same goes in different places in this PR).